docs/getting-started.md

```diff
@@ -4,12 +4,12 @@ title:  Welcome to the Prysmatic Labs Official Documentation Portal
 sidebar_label: Getting started
 ---
 
-![Prysm logo](/img/logo3.png "Prysmatic Labs")
-
 ## What is Prysm?
 
 The [Prysm](https://github.com/prysmaticlabs/prysm) project is a full-featured client for the Ethereum 2.0 network written entirely in the [Go programming language](https://golang.org). Created by [Prysmatic Labs](https://prysmaticlabs.com), Prysm implements the official [Ethereum 2.0 specification](https://github.com/ethereum/eth2.0-specs), which is the product of an ongoing collective research and development effort by various teams across the Ethereum ecosystem including the [Ethereum Foundation](https://ethereum.org).
 
+![Prysm logo](/img/logo3.png "Prysmatic Labs")
```

Comment:
Ahh, I should have mentioned the landing page is an exclusion. Would you mind reverting this one?